samples/cpp/benchmark_app/remote_tensors_filling.cpp

```diff
@@ -11,12 +11,15 @@
 #include <utility>
 #include <vector>
 
-#ifdef HAVE_DEVICE_MEM_SUPPORT
+#ifdef HAVE_GPU_DEVICE_MEM_SUPPORT
 #    include <openvino/runtime/intel_gpu/ocl/ocl.hpp>
 #    include <openvino/runtime/intel_gpu/ocl/ocl_wrapper.hpp>
 #endif
 
-namespace gpu {
+#include <openvino/runtime/intel_npu/level_zero/level_zero.hpp>
+#include <openvino/runtime/intel_npu/remote_properties.hpp>
```

Comment:
Should these two headers be under `ifdef`, or are they always available?
Same for `npu::get_remote_input_tensors()`'s body.

Reply:
This is a good question. From my understanding that part is always available. @ilya-lavrenov could you please help here and confirm if https://github.com/openvinotoolkit/openvino/tree/master/src/inference/include/openvino/runtime/intel_npu will be always available?

Reply:
Yeah, already did that and seems to work. But just wanted another confirmation to be sure that I'm not missing something else.

Reply:
If these headers don't contain other 3rd parties, it's OK to include them unconditionally.